content/developer/howtos/standalone_owl_application.rst

```diff
@@ -0,0 +1,163 @@
+=====================================
+Creating a standalone Owl application
+=====================================
+
+For any number of reasons, you may want to have a standalone Owl application that isn't a part of
+the web client. One example in Odoo is the self-ordering application, that lets customers order
+food from their phone. In this chapter we will go into what's required to achieve something like this.
+
+Overview
+========
+
+To have a standalone Owl app, a few things are required:
+
+- a root component for the application
+- some setup code to start the services and mount the component
+- an assets bundle that contains the setup code
+- an QWeb view that calls the assets bundle
+- a controller that renders the view
+
+1. Root component
+=================
+
+To keep things simple, let's start with a very straightforward component that just renders
+"Hello, World!". This will let us know at a glance if our setup is working.
+
+.. code-block:: js
+
+    /** @odoo-module */
+    import { Component, xml } from "@odoo/owl";
+    class Root extends Component {
+        static template = xml`Hello, World!`;
+    }
+
+.. seealso::
+   Owl components :ref:`reference<frontend/components>`.
+
+2. Setting up the environment and mounting the application
+==========================================================
+
+An Owl application needs an environment, and the Odoo JavaScript framework needs that environment to
+contain the :ref:`services<frontend/services>`.
+Starting the services is also required to load the translations, which we need to do before mounting
+our Owl application, so that we can give Owl a working translation function.
+
+.. code-block:: js
+
+    /** @odoo-module */
+    import { Component, xml, whenReady, App } from "@odoo/owl";
+    import { makeEnv, startServices } from "@web/env";
+    import { templates } from "@web/core/templates";
+    import { _t } from "@web/core/l10n/translation";
+
+    class Root extends Component {
+        static template = xml`Hello, World!`;
+    }
+
+    whenReady().then(async () => {
```

Comment:
i think it would make sense to split the code already in 3 files: `app.js`, `root.js` and `root.xml` to give them a better starting point, with the component definition separate from the setup code